packages/portfolio-api/src/constants.js

```diff
@@ -42,6 +47,30 @@ export const AxelarChain = /** @type {const} */ ({
 });
 harden(AxelarChain);
 
+/**
+ * cf. https://chainlist.org/
+ * XXX this probably belongs in @agoric/orchestration
```

Comment:
I think it already is:

https://github.com/Agoric/agoric-sdk/blob/master/packages/orchestration/src/cctp-chain-info.js#L25-L30

Do you need this change for your PR or is it general cleanup? If it's cleanup we should also update portfolio-deploy to use the same values. And possibly move down to Orchestration. But I think we should defer such cleanup to another PR. This one is already quite large. cc @dckc for his opinion

Reply:
> I do need the `CaipChainIds` that is built from [`Eip155ChainIds`].

I could define `spectrumChainIdsByCluster` redundantly, but that will create drift problems that are better avoided. So I would characterize this as going beyond general cleanup, despite the existence of alternative approaches.